app/models/orphan.rb

```diff
@@ -123,6 +126,17 @@ def current_sponsor
     current_sponsorship.sponsor if sponsored?
   end
 
+  def self.to_csv(records = [])
+    attributes = %w(osra_num full_name father_name date_of_birth gender province_name partner_name 
+      father_is_martyr father_deceased mother_alive priority status sponsorship_status)
+    CSV.generate(headers: true) do |csv|
+      csv << attributes.map(&:titleize)
+      records.each do |orphan|
+        csv << attributes.map { |attr| orphan.send(attr) }
```

Comment:
`public_send` might be better here than `send` which also allows for calls to private/protected methods